orc.js

```diff
@@ -1,42 +1,169 @@
-var app = require('commander');
+var commander = require('commander');
 var git = require('gitty');
 var path = require('path');
-
-// process.stdin.resume();
-// process.stdin.setEncoding('utf8');
-// process.stdin.on('data', function(data) {
-//  process.stdout.write(data);
-// });
+var pushover   = require('pushover');
 
 process.on('SIGINT', function () {
   console.log('Got a SIGINT. Goodbye cruel world');
   process.exit(0);
 });
 
-function _requireCommit() {
+function init(dir, cb) {
+  // TODO: march up to root dir just to make output consistent
+  var repo = git(dir);
+  repo.getBranches(function(err, branches) {
+    if(err) {
+      cb('error getting branches '+err);
+      return;
+    }
+    if (!branches.current) {
+      cb('error: '+repo.name+' doesn\'t appear to be a valid git repo');
+      return;
+    }
+    cb(null, { repo_dir: dir, repo: git(dir)});
+  });
```

Comment:
why the repo: git(dir)? Isn't repo set at the beginning of this function? Shouldn't it be just repo: repo ?